manifest.json

```diff
@@ -1,11 +1,12 @@
 {
     "files": {
-        "index.js": "610cc881f98413c6aa1c7229f87c6beeda43553fa4c5dc53bf4259b514a0eef6",
+        "README.md": "e51994e26e7cee937cf6df94a14783dc4e51be880aaa1d848befef1464878c05",
+        "index.js": "0fd872a2e689b0adf7886303e09483ebdcb56c8b7bfefd76ea3aa7ae4a658ebf",
         "module.json": "632e8ae936ebc7e300590b4c7671c3e4cab3279ee49229d7500e2cb054ca3095"
     },
     "defs": {
         "C_DEL_ITEM": 2,
-		"S_INVEN": [17,18]
```

Comment:
You forgot to add a comma at the end of line here, which will lead to error when reading the json file, as there is still data below it.